eleven-group/pickup
# token management

HOW TO TEST :
before all : make instal
add "|users" in security.yaml line 28 TEMPORARY
get /api/users from api
take one token from one user with isActive = false
copy it
paste it at the {frontUrl}/confirmation/token/{yourcopiedtoken}
a call in the backend should be done
u should be redirected to '/'
